operators/src/processing/point_in_polygon.rs

```diff
@@ -133,6 +133,10 @@ impl PointInPolygonFilterProcessor {
         polygons: &MultiPolygonCollection,
         thread_pool: &ThreadPoolContext,
     ) -> Vec<bool> {
+        if points.is_empty() {
```

Comment:
wouldn't it make more sense to check this before spawning this task in the first place?

Reply:
Yes

11d2c98